html-forms/html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="email" id="email" name="email" />
+      <div>
+        <label for="why-is-coding-cool">Why is coding cool?</label>
+      </div>
+      <select id="why-is-coding-cool">
+        <option value="default">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <p>Favorite Language (choose one)</p>
+      <p>
+        <input type="radio" id="HTML" name="favorite-language" value="html" />
+        <label for="CSS">HTML</label>
+        <input type="radio" id="CSS" name="favorite-language" value="css" />
+        <label for="HTML">CSS</label>
```

Comment:
make sure these labels are matching/pointing to the right inputs

Reply:
![Capture](https://github.com/aehrenkranz/m1-0723-code-solutions/assets/130402253/1860785d-dc77-44ee-90e6-33a410176469)

Updated label tags to correct IDs